### PR TITLE
Added python reference app with Auth0 and Added metadata information in AppInformation.json

### PR DESCRIPTION
**Pull request by Affinidi-Grajesh**
Added Python Reference app with Auth0 integration
1. Flask
2. FastAPI
3. Starlette

Also, Added metadata for all reference application in AppInformation.json.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +35,11 @@
     "start:affinidi-django-authlib": "python3 samples/affinidi-django-authlib/manage.py migrate && python3 samples/affinidi-django-authlib/manage.py runserver 8000",
     "start:auth0-django-authlib": "python3 samples/auth0-django-authlib/manage.py migrate && python3 samples/auth0-django-authlib/manage.py runserver 8001",
     "start:affinidi-flask-authlib": "cd samples/affinidi-flask-authlib && python3 -m flask run -p 8100",
+    "start:auth0-flask-authlib": "cd samples/auth0-flask-authlib && python3 -m flask run -p 8101",
     "start:affinidi-fastapi-authlib": "cd samples/affinidi-fastapi-authlib && python3 app.py",
```
Comment (pulkitb2) [inline package.json:39]: what are port numbers for fastapi and startlette? Are they fixed?

File: reference-app-affinidi-vault.sln [hunk 2]
```diff
@@ -0,0 +1,42 @@
+﻿
```
Comment (pulkitb2) [inline reference-app-affinidi-vault.sln:1]: What is purpose of this file?

File: package.json (hunk 1 above)
Comment (Affinidi-Grajesh) [reply package.json:39]: yes, they are fixed with 8201 and 8301 respectively.

File: reference-app-affinidi-vault.sln (hunk 2 above)
Comment (Affinidi-Grajesh) [reply reference-app-affinidi-vault.sln:1]: I didn't see this file in the repo.

Comment (pulkitb2) [reply reference-app-affinidi-vault.sln:1]: okay, but why is it added?

**Review by rolstan-affinidi — DISMISSED** [review]

File: reference-app-affinidi-vault.sln (hunk 2 above)
Comment (sanjayK-aff) [reply reference-app-affinidi-vault.sln]: this is the solution file for dotnet project, not needed but autogenerates if opened in Visual studio or supported editors. 
REMOVED.

**Review by rolstan-affinidi — APPROVED** [review]

**Review by pulkitb2 — APPROVED** [review]

**Pull request merged**